executors/cmdexec/block_volume.go

```diff
@@ -337,8 +337,11 @@ func (s *CmdExecutor) BlockVolumeExpand(host string, blockHostingVolumeName stri
 			output)
 	} else if blockVolumeExpand.Result == "FAIL" {
 		// the fail flag was set in the output json
-		err = fmt.Errorf("Failed to expand block volume: %v",
-			blockVolumeExpand.ErrMsg)
+		if blockVolumeExpand.ErrMsg != "" {
+			err = fmt.Errorf("%v, please check logs, and retry again.", blockVolumeExpand.ErrMsg)
```

Comment:
The wording here strikes me as a bit odd. Perhaps:
```
Failed to expand block volume: %v (see logs for details, and retry the operation)
&
Failed to expand block volume (see logs for details, and retry the operation)
```

Reply:
ok